defaults/main.yml

```diff
@@ -32,4 +32,24 @@ dell_dsu_changed_exit_codes:
   - 8 # Reboot is required for the update to be completed successfully
 dell_dsu_not_changed_exit_codes:
   - 34 # There were no updates found for the system.
+
+firmeware_components:
```

Comment:
While this doesn't apply to everything, I think this is a case where we need to be pedantic.

`firmeware` -> `firmware`

Reply:
I agree. I will update it